operators/advanced_network/advanced_network/manager.cpp

```diff
@@ -112,9 +126,35 @@ ManagerType ManagerFactory::get_manager_type(const Config& config) {
 
 template ManagerType ManagerFactory::get_manager_type<Config>(const Config&);
 
+size_t Manager::get_alignment(MemoryKind kind) {
+  switch (kind) {
+    case MemoryKind::HOST:
+    case MemoryKind::HOST_PINNED:
+    case MemoryKind::HUGE:
+      return 128;
+    case MemoryKind::DEVICE:
+      return 256;
+    default:
+      return 128;
+  }
+}
```

Comment:
where do these values come from? I see only used for RDMA? Can you add some description?